tests/test_validate_source_skill_lineage.py

```
@@ -131,6 +131,123 @@ def load_refresher():
+                [
+                    sys.executable,
+                    str(REFRESHER),
+                    "capture-host",
+                    str(REPOSITORY),
+                    "--private-input",
+                    str(temporary / "private-input.json"),
+                ],
+            )
+            for command in commands:
+                with self.subTest(command=command[2]):
+                    completed = subprocess.run(
```

Ruff (0.16.3)

[error] `subprocess` call: check for execution of untrusted input

(S603)